common/src/app/common/types/shape/layout.cljc

```diff
@@ -1622,10 +1622,14 @@
 (defn remap-grid-cells
   "Remaps the shapes ids inside the cells"
   [shape ids-map]
-  (let [do-remap-cells
+  (let [do-map-shape
```

Comment:
Can we change this naming, `do-something` usually means side effect, i think this can be `resolve-shape-id` or `lookup-shape-id`.

And `do-remap-cells` can be `remap-cells`

Reply:
OK, changed.